modules/messages/client/views/thread.client.view.html

```diff
@@ -105,7 +105,22 @@
       </threads>
       <!-- /Thread -->
 
+      <!-- quickreply -->
+      <div class="btn-toolbar" id="message-quick-reply" ng-hide="thread.hideQuickReplyPanel">
+          <button class="btn btn-success icon-ok" ng-click="thread.sendMessage('<b><i>'+app.appSettings.messages.msgCanHost+'</i></b>');">
+            {{app.appSettings.messages.msgCanHost}}
+          </button>
+          <button class="btn btn-danger icon-minus-squared-alt" ng-click="thread.sendMessage('<b><i>'+app.appSettings.messages.msgCannotHost+'</i></b>');">
+            {{app.appSettings.messages.msgCannotHost}}
+          </button>
+          <button class="btn btn-info icon-reply" ng-click="thread.hideQuickReplyPanel=true;">
+            {{app.appSettings.messages.msgWriteBack}}
```

Comment:
Use angular [one time binding](https://docs.angularjs.org/guide/expression#one-time-binding) (`::`) here:
```js
{{::app.appSettings.messages.msgWriteBack}}
```
...since those labels don't change and no need to watch for changes.